config/checkstyle-resources-suppressions.xml

```diff
@@ -256,6 +256,8 @@
     files="[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodPublicOnly1\.java"/>
   <suppress checks="FileLength"
     files="[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]javadoc[\\/]javadocmethod[\\/]InputJavadocMethodProtectedScopeJavadoc\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]test[\\/]resources[\\/]com[\\/]puppycrawl[\\/]tools[\\/]checkstyle[\\/]checks[\\/]javadoc[\\/]javadocparagraph[\\/]InputJavadocParagraphIncorrect\.java"/>
```

Comment:
Please remove `with no space after.` to keep violation short.
It is not very critical part of message

Reply:
done

Reply:
Please remove this suppression

Reply:
can't do this. The input file is longer than 120 lines, it'll start giving Filelength error:

```
[ERROR] [checkstyle] [ERROR] C:\OpenSource\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\javadoc\javadocparagraph\InputJavadocParagraphIncorrect.java:1: File length is 138 lines (max allowed is 120). [FileLength]
```